pkg/db/v1beta1/postgres/postgres_test.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2022 The Kubeflow Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package postgres
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	sqlmock "github.com/DATA-DOG/go-sqlmock"
+	_ "github.com/lib/pq"
+
+	api_pb "github.com/kubeflow/katib/pkg/apis/manager/v1beta1"
+	"github.com/kubeflow/katib/pkg/db/v1beta1/common"
+)
+
+var dbInterface common.KatibDBInterface
+var mock sqlmock.Sqlmock
+
+func TestMain(m *testing.M) {
+	db, sm, err := sqlmock.New()
+	mock = sm
+	if err != nil {
+		fmt.Printf("error opening db: %v\n", err)
+		os.Exit(1)
+	}
+	dbInterface, err = NewWithSQLConn(db)
+	if err != nil {
+		fmt.Printf("error NewWithSQLConn: %v\n", err)
+	}
+	mock.ExpectExec("CREATE TABLE IF NOT EXISTS observation_logs").WithArgs().WillReturnResult(sqlmock.NewResult(1, 1))
+	dbInterface.DBInit()
+	mock.ExpectExec("SELECT 1").WithArgs().WillReturnResult(sqlmock.NewResult(1, 1))
+	err = dbInterface.SelectOne()
+	if err != nil {
+		fmt.Printf("error `SELECT 1` probing: %v\n", err)
+	}
+	os.Exit(m.Run())
```

Comment:
```suggestion
	m.Run()
```
Since Go v1.15, we do not need to call `os.Exit`.

> A TestMain function is no longer required to call os.Exit. If a TestMain function returns, the test binary will call os.Exit with the value returned by m.Run.

https://go.dev/doc/go1.15

Reply:
Wow Thanks for sharing :)